cert/ca.go

```diff
@@ -109,7 +110,7 @@ func PktTox509(pktCom []byte, caBytes []byte, caPkSk *ecdsa.PrivateKey, required
 			Critical: false,
 			Value:    []byte(oidcIssuer),
 		}},
-		SubjectKeyId: []byte(pktCom),
+		SubjectKeyId: []byte(base64.URLEncoding.EncodeToString(pktJson)),
```

Comment:
I was thinking about your argument that everything should be base64 URI encoded. I want to have a common place where all base64 encoding happens so that all the code is using the same encoder and we can switch it easily if we need to.

Could you which this to use `util.Base64EncodeForJWT`
https://github.com/openpubkey/openpubkey/blob/main/util/base64.go#L15C6-L15C24